examples/extensions/commands/custom_commands.rst

```diff
@@ -0,0 +1,174 @@
+.. _examples_extensions_custom_commands:
+
+
+Custom commands
+=================
+
+Since Conan 2.0, it's so easy to create your own Conan commands. These are some of the rules/tips to create custom commands:
+
+* All the custom commands must be located under the ``[YOUR_CONAN_HOME]/extensions/commands/`` folder.
+    * It can be created another folders layer to act like ``scope`` for the commands under it, e.g., ``[YOUR_CONAN_HOME]/extensions/commands/myscope/``.
```

Comment:
`It can be created another folders layer to act like ``scope`` for the commands under it` - IMO the wording is a bit unclear, I would split the sentence, or rephrase.